core/src/main/java/org/apache/carbondata/core/view/MVProvider.java

```diff
@@ -152,39 +130,33 @@ public MVSchema getSchema(MVManager viewManager,
 
   void saveSchema(MVManager viewManager, String databaseName,
                   MVSchema viewSchema) throws IOException {
-    SchemaProvider schemaProvider = this.getSchemaProvider(databaseName);
+    SchemaProvider schemaProvider = this.getSchemaProvider(viewManager, databaseName);
     if (schemaProvider == null) {
       throw new IOException("Database [" + databaseName + "] is not found.");
     }
     schemaProvider.saveSchema(viewManager, viewSchema);
   }
 
-  public void dropSchema(String databaseName, String viewName)
-      throws IOException {
-    SchemaProvider schemaProvider = this.getSchemaProvider(databaseName);
+  public void dropSchema(MVManager viewManager, String databaseName,
```

Comment:
only this change required `SchemaProvider schemaProvider = this.getSchemaProvider(viewManager, databaseName);`, please  revert method signature change

Reply:
Argument [viewManager] added for this method